src/Infrastructure/Migrations/Version20230206081319.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\PriceHistoryPlugin\Infrastructure\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20230206081319 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add ChannelPricing::lowestPriceBeforeDiscount';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE sylius_channel_pricing ADD lowestPriceBeforeDiscount INT DEFAULT NULL');
```

Comment:
Column names have to be in snake case so `lowest_price_before_discount` instead of `lowestPriceBeforeDiscount`.
It should be fixed in this migration and in the mapping in the trait.